### PR TITLE
english.php, both

### PR DESCRIPTION
**Pull request by torvista**
Capitalisation of the subjects in the item listing in prev-next
other minor stuff as per commit listing

### CODE REVIEW

File: admin/includes/languages/english.php [hunk 1]
```diff
@@ -698,5 +698,4 @@ function zen_date_raw($date, $reverse = false) {
 ///////////////////////////////////////////////////////////
 // include additional files:
   require(DIR_WS_LANGUAGES . $_SESSION['language'] . "/" . FILENAME_EMAIL_EXTRAS);
-  include(zen_get_file_directory(DIR_FS_CATALOG_LANGUAGES . $_SESSION['language'] . '/', FILENAME_OTHER_IMAGES_NAMES, 'false'));
-  
\ No newline at end of file
+  include(zen_get_file_directory(DIR_FS_CATALOG_LANGUAGES . $_SESSION['language'] . '/', FILENAME_OTHER_IMAGES_NAMES));
```
Comment (mc12345678) [inline admin/includes/languages/english.php:701]: I thought that somewhere in "here" (GitHub) that I had commented/asked about this use of the default value in the call to the function and had received a response indicating that it was desirable to maintain the `'false'` statement... I can't seem to find that (or perhaps I'm thinking of another similar usage). So I ask what is the background in removing it from here? Further, is a similar change going to be pushed (pulled? :)) to all such uses?

Comment (torvista) [reply admin/includes/languages/english.php:701]: No concrete reason, just as it is the default.

File: admin/includes/languages/english.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
 define('HEADER_LOGO_IMAGE', 'logo.gif');
 
 // look in your $PATH_LOCALE/locale directory for available locales..
-$locales = array('en_US', 'en_US.utf8', 'en', 'English_United States.1252');
```
Comment (scottcwilson) [inline admin/includes/languages/english.php:23]: why are we changing array() to [] ?

Comment (torvista) [reply admin/includes/languages/english.php:23]: I've been changing to the short form on files I modify, I understand it's preferred.

Comment (scottcwilson) [reply admin/includes/languages/english.php:23]: ^^^^ @zcwilt @drbyte please confirm and I will update the docs.

Comment (zcwilt) [reply admin/includes/languages/english.php:23]: definitely prefer the short form.

**Pull request merged**